### DISCUSSION TITLE
Dev Tools - Getting Maximum execution time of 30 seconds exceeded error after upgrade to vr. 3.0.4

### DISCUSSION THREAD

**Discussion by webteamseo** [Laravel-Backpack/community-forum#831] [Q&A (Help)] [answered]
Hi,
here is the info:
https://flareapp.io/share/W7zwOY25

When I try to change the max_execution_time to 120 it still doesn't work.

I've also removed all paths from config/devtools.php.

If I downgrade to version 2.0.X it works, but not usable (buttons are misplaced and checkboxes cannot be checked).

I am on Laravel 10.43 - recently upgraded
Backpack 6 - recently upgraded
Devtools 3.0.4
PHP 8.2.1
Macbook pro with Homestead Laravel with VM with 4 cpu's and 8 GB Memory allocated.

Any help would be really appriciated.

**Comment by jcastroa87** [comment]
Hello @webteamseo

Have you been able to try a clean install? i just installed and there is no problem with the operation:

```
### PHP VERSION:
PHP 8.2.11 (cli) (built: Oct  6 2023 09:46:55) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.11, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.11, Copyright (c), by Zend Technologies

### LARAVEL VERSION:
10.43.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/activity-log: 2.0.1
backpack/basset: 1.2.2
backpack/crud: 6.6.1
backpack/devtools: 3.0.4
backpack/generators: v4.0.3
backpack/permissionmanager: 7.1.1
backpack/pro: 2.0.21
backpack/theme-coreuiv2: 1.2.2
backpack/theme-coreuiv4: 1.1.1
backpack/theme-tabler: 1.2.0
```

There is the version i try.

Cheers.

**Comment by webteamseo** [comment]
Hi, I didn't run it on clean install. What I am trying to understand if it is possible that my project is too big so version 3.0.4 takes too much to load. I tried changing the devtools.php under config, to remove paths, but still this issue persist.

**Comment by webteamseo** [comment]


**Reply by webteamseo** [reply]
@tabacitu, sorry for taggin you (I hope this is ok). Can anyone please help? or just reply? I've made such a beautiful video :) and it's been a week now since I've open the discussion.
Thanks

**Comment by pxpm** [comment]
Sorry for taking time to get back here and for the video. 
I've just assigned my self and I am trying to reproduce the issue. 

I will come back with news or with additional queries, like "Now, please do video a reproducing the bug, but you need to sing the problem like you have 99". hehe 😺 

Cheers, and sorry again 🙏

**Comment by pxpm** [comment, marked as answer]
So, I have good news, and bad news. 
Good news is that I couldn't reproduce the issue. Bad news, is that I recall seeing those broken things before we launched v3 of devtools. 

I've just installed the tool now:
![image](https://github.com/Laravel-Backpack/community-forum/assets/7188159/f9b75544-23ba-40fc-bd36-af3b92fbb7cc)

**Question 1**: can you confirm that when you install devtools you get the same versions of those packages ? 

Here are the coreui-v4 and theme-tabler working as expected:
![image](https://github.com/Laravel-Backpack/community-forum/assets/7188159/2933b77d-8fd4-4bb0-b383-654fad7ba917)
![image](https://github.com/Laravel-Backpack/community-forum/assets/7188159/5d380fed-f3c7-45fc-91de-71c586bd3e06)

So .. I suspected of 2 things:
1 - You may have the files overwritten in your `resources` folder. If that's the case, you are not getting the updated versions of the files from the vendor.
2 - Your css/js/views are cached and you are not loading the correct files when changing themes. 

For 1, I would just look into your `resources/views/vendor/backpack` to check if you have anything related to devtools there. For 2, after changing the themes I would try to run: `php artisan cache:clear && php artisan view:clear`. If you have basset enabled (you probably should disable it while developing), run also `php artisan basset:clear && php artisan basset:cache`. 

To disable basset in development add to your .env: `BASSET_DEV_MODE=true` and then run a `php artisan basset:clear` to make sure any artifacts of basset are cleared. 

Let me know how it goes. 

Cheers

**Comment by webteamseo** [comment]
Yes! it was worth waiting :) it works now. The problem was folder named devtools under resources/views/vendor/backpack
Thank you very much for your help.

**Reply by pxpm** [reply]
Once again, sorry this took a little bit longer than what we all would like, but very happy we were able to quickly find the issue 🙏 

If you have other questions/suggestions feel free to open other discussions 🙏 

Cheers
